src/jaxoplanet/orbits/keplerian.py

```diff
@@ -80,14 +110,18 @@ def from_orbital_properties(
 
         radius = 1.0 * ureg.R_sun if radius is None else radius
 
-        mass = semimajor**3 / (ureg.gravitational_constant * period**2)
+        mass = 4 * jnp.pi**2 * semimajor**3 / (ureg.gravitational_constant * period**2)
```

Comment:
For this one, can you add a test to `tests/orbits/keplerian_test.py`? I think the test from the tutorial (creating the Sun from Earth's orbit) would be a good one!

Reply:
I can do, however, I'm getting a slight (perhaps rounding?) error when doing that simple example:
```
Sun = Central.from_orbital_properties(period=1.0 * ureg.yr, semimajor=1.0 * ureg.au, radius=1.0 * ureg.R_sun, body_mass=1.0 * ureg.M_earth)
```
```
Central(
  mass=<Quantity(1.00003477, 'M_sun')>,
  radius=<Quantity(1.0, 'R_sun')>,
  density=<Quantity(0.238740715, 'M_sun / R_sun ** 3')>
)
```

Reply:
Maybe you're missing the jax config option to work in float64 (doubles) instead of the default float32?

[https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#double-64bit-precision](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#double-64bit-precision)

Reply:
Good suggestion, @soichiro-hattori! This result does look ok to me for `float32`. I think it's nice to make sure that the tests can pass at single precision, so we should be able to use [`jaxoplanet.test_utils.assert_all_close`](https://github.com/exoplanet-dev/jaxoplanet/blob/f7f5a1d1e3e8bcab2f6cbb28b7373b63ed80a64d/src/jaxoplanet/test_utils.py) to implement the test taking the precision into account.

Reply:
Hmm I've included:
```from jax import config
config.update("jax_enable_x64", True)
```
at the start of a notebook and I get the same issue, and this error when I try `jaxoplanet.test_utils.assert_allclose`:

```assert_allclose(Sun.mass, 1.0 * ureg.M_sun)
AssertionError: 
Not equal to tolerance rtol=5e-07, atol=1e-15

Mismatched elements: 1 / 1 (100%)
Max absolute difference: 3.47703597e-05
Max relative difference: 3.47703597e-05
 x: array(1.000035)
 y: array(1.)
```

Reply:
@catrionamurray: I think I figured this out! The `ureg.yr = 365.25 day` is a Solar/Tropical year when we actually want (I think) the sidereal year of **365.2563 days**. Passing the sidereal year to `period` (either in units of days or years) makes the assertion test work for both single and double precision (as an aside 365.256 days will work for single precision but fail for double precision)! I also think we want to be using `jaxoplanet.test_utils.assert_quantity_allclose()` since these are `pint.Quantity` objects?

Reply:
Nice finding! I'd rather not make this redefinition, noting that `astropy.units` also has the same behavior:

```python
>>> import numpy as np
>>> from astropy import units as u, constants as c
>>> (4 * np.pi**2 * (1*u.au)**3 / (c.G * (1 * u.yr)**2)).to(u.M_sun)
<Quantity 1.00003777 solMass>
```

I'd propose that we just add a test with an appropriately generous tolerance. Something like:

```python
>>> from jaxoplanet.test_utils import assert_quantity_allclose
>>> sun = Central.from_orbital_properties(
...     period=1.0 * ureg.yr,
...     semimajor=1.0 * ureg.au,
...     radius=1.0 * ureg.R_sun,
...     body_mass=1.0 * ureg.M_earth,
... )
>>> assert_quantity_allclose(sun.mass, 1.0 * ureg.M_sun, atol=1e-5)
```